docs/docsite/rst/porting_guides/porting_guide_2.9.rst

```diff
@@ -143,8 +143,116 @@ Noteworthy module changes
 Plugins
 =======
 
-No notable changes
+AWS Inventory Plugin
```

Comment:
Did this actually happen in 2.9 (since it's already branched) or does this change need to go into `porting_guide_2.10.rst`?

Reply:
@gundalow  `iam_role_arn` yes but I'm not sure we need these docs if we're not taking the additional filter features.